pkg/monitorsformatter/summaryFormatter.go

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright contributors to the Galasa project
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package monitorsformatter
+
+import (
+	"strconv"
+	"strings"
+
+	"github.com/galasa-dev/cli/pkg/galasaapi"
+	"github.com/galasa-dev/cli/pkg/utils"
+)
+
+// -----------------------------------------------------
+// Summary format.
+const (
+	SUMMARY_FORMATTER_NAME = "summary"
+)
+
+type MonitorsSummaryFormatter struct {
+}
+
+func NewMonitorsSummaryFormatter() MonitorsFormatter {
+	return new(MonitorsSummaryFormatter)
+}
+
+func (*MonitorsSummaryFormatter) GetName() string {
+	return SUMMARY_FORMATTER_NAME
+}
+
+func (*MonitorsSummaryFormatter) FormatMonitors(monitors []galasaapi.GalasaMonitor) (string, error) {
+	var result string
+	var err error = nil
+	buff := strings.Builder{}
+	total := len(monitors)
+
+	if total > 0 {
+		var table [][]string
+
+		var headers = []string{
+			HEADER_MONITOR_NAME,
+			HEADER_MONITOR_KIND,
+			HEADER_MONITOR_IS_ENABLED,
```

Comment:
Exposing the description field here might be good ? even if it is cropped when too long?

Reply:
I agree that the description field should be added, but at the moment the description isn't settable and isn't stored anywhere so will leave an empty space in the output if we were to add the header in now. I think it would be better to add the description into the output once it can be set.